components/myft-concept-button/main.scss

```
@@ -0,0 +1,228 @@
+@import '@financial-times/o-typography/main';
+
+$myft-lozenge-themes: (
```

```suggestion
/// @access private
$_myft-lozenge-themes: (
```

Could we make this private with a leading `_`? Same for any variable/function/mixin that we don't expect the a user to call directly? It'll make it easier to refactor. For example easier to switch to design tokens without a breaking, in the case of the theme variable.

---

yes we should do that.